### DISCUSSION TITLE
[General] Optimism Superchain's Base on profit after Dencun

### DISCUSSION THREAD

**Discussion by konityan** [ethereum-optimism/developers#595] [General]
### Discussion Topic

Below is an article about the decrease in PROFIT after the Dencun update for Base.
If you read the article carefully, 
you will see that the PROFIT has decreased compared to the last 4 weeks since the Dencun update.

![image](https://github.com/user-attachments/assets/2ca3241b-a4a2-418f-9037-9cdde989f411)
https://blockworks.co/news/base-profit-down-while-transactions-up

Why does the PROFIT increase only for 4 weeks after Dencun, 
and then the PROFIT decreases while the number of transactions is steadily increasing after that?

![image](https://github.com/user-attachments/assets/426d1292-afeb-4c4f-b731-d758167c914d)
https://dune.com/queries/2838842/4740735

Please let me know if you are an expert in this area.

**Comment by hash1go** [comment]
Hello.
The high profits for Base before and after the Dencun upgrade were due to the block size being small relative to demand.

The OP Stack fee structure has a two-tier structure, and since L1 on-chain costs are passed on to all transaction submitters based on a [calculation formula](https://docs.optimism.io/stack/transactions/fees#formula), the Dencun upgrade does not directly affect OP Stack profits.
However, the Dencun upgrade reduced the on-chain costs being passed on, so even with high base fees + priority fees, users had an incentive to use block space, which undoubtedly became an indirect factor generating significant profits.

Base announced an expansion of block size in response to increased demand for block space, and this expansion is still ongoing. Since L2 execution fees are based on EIP-1559, the expansion of block size is a factor in reducing profits, and profits appear to be continuously decreasing.
https://x.com/growthepie_eth/status/1816443414189117754
https://base.mirror.xyz/6NDvVKw8x5obo3h1OgQx582hF73m81CnJabgbRneW2Q
